views/layout.pug

```diff
@@ -1,7 +1,8 @@
 doctype html
 html
 	head
-		title= title
```

Comment:
Is it normal that the `=` disappeared?

Reply:
Yep, firstly, it was not used and secondly as we only have one page, there is no need to be dynamic